### PR TITLE
Gitpod

### PR DESCRIPTION
**Pull request by shaal**
## Description
The purpose of this PR is to fully automate the dev setup of Drupal Rector with Gitpod, providing anyone with a pre-built, browser-based development environment in one click:

[Open in Gitpod](https://gitpod.io/from-referrer/)

Gitpod provides automated dev environments in the cloud (free to use for public repos).
By adding `.gitpod.yml` file to this repo, we will improve developer experience when working on Drupal Rector repo, without modifying the current way developers working on it.

When clicking on "Open in Gitpod" button, a user gets a whole development environment in their browser.
Eliminating the need for each contributor to figure out how to setup a development environment. 

This is an additive change, so it wouldn't affect anyone who wants to continue work on their local machine, but will lower the barriers for anyone who wants to contribute.

In order to automatically generate Gitpod prebuilds, this Github app https://github.com/apps/gitpod-io should be authorized to run on this repo.

## To Test
* Click on [Open in Gitpod](https://gitpod.io/from-referrer/)
* If a login screen appears, authorize Gitpod with Github
* Wait for VSCode interface to load
* In the terminal panel type -
```
cd drupal-website
composer require drupal/pathauto
vendor/bin/rector process web/modules/contrib/pathauto --dry-run
```
- Confirm Drupal Rector works as expected.

## Drupal.org issue
https://www.drupal.org/node/3227040

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,11 +150,25 @@ Thanks for your interest in contributing!
 
 Our goal is to make contributing to this project easy for people. While we've made certain architectural decisions here to hopefully achieve that goal, it's a work in progress and feedback is appreciated.
 
-### Development environment
+## Development environment
+
+We recommend using Gitpod, a full development environment in the cloud.
```
Comment (agentrickard) [inline README.md]: I don't think this language is agreed upon. "You may use Gitpod" is likely better.

Comment (shaal) [reply README.md]:
```suggestion
You may use Gitpod, a full development environment in the cloud.
```

**Comment by mglaman** [comment]
In my opinion this should go to the drupal_rector_sandbox and not this repo. Just because it has a lot of extras. Technically developing drupal-rector just requires a `git clone` but this GitPod setup does much more.

**Comment by mglaman** [comment]
Also, just linking `https://gitpod.io/#/github.com/palantirnet/drupal-rector` is very sufficient.

**Comment by agentrickard** [comment]
Agreed. Let's move this to the sandbox project.

**Comment by RobinHoutevelts** [comment]
Has this been moved then? I can't seem to find anything GitPod related in https://github.com/palantirnet/drupal-rector-sandbox

**Pull request closed without merging**